daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -31,7 +31,7 @@ import java.nio.charset.StandardCharsets
 import java.util.Scanner
 import java.util.concurrent.Executors
 import javax.xml.parsers.DocumentBuilderFactory
-import javax.xml.transform.TransformerFactory
+import javax.xml.transform.{ TransformerFactory, TransformerException }
```

Comment:
Can you split these to two lines? I think we've started moving away from multiple imports on a single line.